app/script/auth/module/action/creator/AuthActionCreator.js

```diff
@@ -147,3 +153,25 @@ export function enterTeamCreationFlow() {
 export function enterPersonalCreationFlow() {
   return {type: ENTER_PERSONAL_CREATION_FLOW};
 }
+
+export function enterGenericInviteCreationFlow() {
+  return {type: ENTER_GENERIC_INVITATION_FLOW};
+}
+
+export function enterPersonalInvitationCreationFlow() {
+  return {type: ENTER_PERSONAL_INVITATION_FLOW};
+}
+
+export const startGetInvitationFromCode = () => ({
+  type: GET_INVITATION_FROM_CODE_START,
+});
```

Comment:
A few arrow functions and a few others? Shouldn't we align these?

Reply:
I took care of those in another PR.